web_monitoring/cli/cli.py

```diff
@@ -338,9 +338,14 @@ def format_memento(self, memento, cdx_record, maintainers, tags):
             if k.startswith(prefix)
         }
 
+        # Parse media type information.
+        media_type, *media_params = memento.headers.get('content-type', '').split(';')
+        # Clean up whitespace, remove empty parameters, etc.
+        stripped_params = (param.strip() for param in media_params)
+        media_params = [param for param in stripped_params if param]
+        media_type_parameters = '; '.join(media_params)
```

Comment:
Future note: I think we may want to push this parsing down into the Wayback package. Then we might have something like:

```python
media_type = memento.media.type
media_type_parameters = str(memento.media.parameters)
```